docs/topics/vscode-extension-reviewing-projects.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-reviewing-projects_{context}']
+
+= Reviewing and resolving issues with the {ProductName} extension
+
+You can review and resolve issues identified by the {ProductName} extension in the Issue Run configuration pane.
+
+== Reviewing issues
+
+The {ProductName} extension uses icons to indicate the severity of an issue and wherher or not an issue can be solved by applying Quick Fixes (automatic code replacement).
+
+.Procedure
+
+. Select a run configuration directory on the Run configuration pane and expand its folders to view the *Hints* generated for each application file.
+. Select a hint to view the source code.
+. Right-click a hint and select *View Details* to view the *Rule ID* and other information.
++
+To the left of each hint is an icon showing the severity of the issue and an indication if it is possible to apply Quick Fixes to the issue.
```

Comment:
Suggestion: The icon beside the hint displays the issue severity and whether a Quick Fix is available.